src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,98 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    private static final int INITIAL_CAPACITY = 16;
+    private static final int SIZE_MULTIPLIER = 2;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private int currentCapacity;
+    private int threshold;
```

Comment:
Mentors says "Do not create redundant fields! Can we user table.length instead?", but for me it's ok

Reply:
agree, I will change it